frontend/src/metabase/visualizer/utils/split-series.ts

```diff
@@ -2,11 +2,30 @@ import _ from "underscore";
 
 import { isNotNull } from "metabase/lib/types";
 import { isCartesianChart } from "metabase/visualizations";
-import type { RawSeries } from "metabase-types/api";
+import type { RawSeries, VisualizationSettings } from "metabase-types/api";
 import type { VisualizerColumnValueSource } from "metabase-types/store/visualizer";
 
 import { isDataSourceNameRef } from "./data-source";
 
+export function shouldSplitVisualizerSeries(
```

Comment:
Could you add a comment to describe under what circumstances this would return true and what "splitting visualizer series" entails?